components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -638,8 +638,11 @@ def getJSToNativeConversionInfo(type, descriptorProvider, failureCode=None,
     # Note that we can't just set failureCode to exceptionCode, because setting
     # failureCode will prevent pending exceptions from being set in cases when
     # they really should be!
+    unhandledExceptionCode = exceptionCode
     if exceptionCode is None:
-        exceptionCode = "return false;"
+        exceptionCode = 'return false;'
+        unhandledExceptionCode = 'debug!("Unhandled exception error"); return false;'
```

Comment:
The point is that we don't set any exception and the script will abort silently... I think it's worth to print a message in this case, but if you think it's not I can remove it.